docker.go

```diff
@@ -79,16 +94,36 @@ func (container *Container) Volume(hostPath, containerPath, mode string) *Contai
 func NewDockerContainer() *Container {
 	container := new(Container)
 	container.Type = "DOCKER"
-	container.Docker = &Docker{
-		Image:        "",
-		Network:      "BRIDGE",
-		PortMappings: make([]*PortMapping, 0),
-		Parameters:   make([]*Parameters, 0),
-	}
-	container.Volumes = make([]*Volume, 0)
+	container.EmptyVolumes()
+	container.Docker = &Docker{}
+	container.Docker.Container("").
+		Bridged().
+		SetPrivileged(false).
```

Comment:
Agreed: A breaking change isn't ideal but for the better in this case as no default network should really be set.

One thing we could do to make people aware of the behavior shift is to mention the change in the commit message, maybe along with a **NOTE:** prefix.